### ISSUE TITLE
Unable to resolve peer dependency with latest version of React.

### ISSUE THREAD

**Issue by edsulaiman** [avkonst/hookstate#154]
## Describe the bug
Error to install dependency in a new react app.

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: client-react@0.1.0
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   react@"^17.0.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.6" from @hookstate/core@3.0.6
npm ERR! node_modules/@hookstate/core
npm ERR!   @hookstate/core@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
```

## To Reproduce

- Create a new app with ```create-react-app```
- Run command ```npm install --save @hookstate/core```
- Installation will be rejected.

**Comment by avkonst** [comment]
Please see:
https://github.com/avkonst/hookstate/pull/126#issuecomment-793027790

**Comment by avkonst** [comment]
Support for React 17 peer dependency has been released in 3.0.7
